recipes/python-ray/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "ray" %}
+{% set version = "0.6.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ray-project/{{ name }}/archive/ray-{{ version }}.tar.gz
```

Comment:
I think you should use `https://github.com/ray-project/{{ name }}/archive/{{ name }}-{{ version }}.tar.gz`

Reply:
Why do not you use https://pypi.org/project/ray/ ?

Reply:
That was the first thing I tried, but for some reason `conda skeleton pypi ray` fails with “url not found” (I think it’s because they only put the wheels, not the source)

Reply:
I guess that is the same issue as discussed in https://github.com/pypa/warehouse/issues/5579#issuecomment-484469344

Reply:
I'd be very surprised if the "url not found" message is at all related to the issue you mention; it displays a different error - "bad gateway" , a 504.

I think it is not found because no .tar.gz is pulished to PyPI. A list of files published is as <https://pypi.org/project/ray/#files> and only includes files with a .whl extension

Reply:
Yeah, I think the reason is no .tar.gz files. But I'm honestly at a loss on how to "direct" the build to the setup.py file inside the python folder from the project (and if that would be enough to build it)